context/app/static/js/index.jsx

```diff
@@ -9,11 +9,18 @@ import App from './App';
 import Footer from './Footer';
 import Header from './Header';
 
-if (window.location.href.search('/browse/') > -1) {
+const urlPath = window.location.pathname;
+
+if (urlPath.indexOf('/browse') > -1) {
```

Comment:
Seems to overlap with Routes.jsx -- Are they both necessary?

Reply:
Both are necessary so long as there are situations, now just the error pages, where we need to only attach the header/footer and not any content.